app/models.py

```diff
@@ -1157,3 +1157,17 @@ def is_readable_by(self, user_or_token):
 TokenACL.__doc__ = 'Join table mapping Tokens to ACLs'
 UserRole = join_model('user_roles', User, Role)
 UserRole.__doc__ = 'Join table mapping Users to Roles.'
+
+
+class CronJobRun(Base):
+    """A record of a run (or attempted run) of a cron job."""
+    script = sa.Column(
+        sa.String,
+        nullable=False,
+        doc="Name of script being run.",
+    )
+    exit_status = sa.Column(
```

Comment:
Should we also log stdout/stderr?

Reply:
@stefanv  Good call - just pushed up that change

Reply:
I tested that it captures both stdout and stderr